client/my-sites/email/email-providers-stacked-comparison/provider-cards/utils.tsx

```diff
@@ -0,0 +1,149 @@
+		components: {
+			price: (
+				<span className={ formattedPriceClassName }>
+					{ formatCurrency( cost ?? 0, currencyCode ) }
+				</span>
+			),
+			termLength: <span className={ termLengthClassName }>/{ termLength }</span>,
+		},
+		comment:
+			'{{price/}} is the formatted price, e.g. $20' +
+			'{{termLength/}} is already translated and it is either annually or monthly',
+	} );
+};
+
+export const AddToCartAndCheckout = (
+	shoppingCartManager: any,
+	cartItem: RequestCartProduct,
+	productList: any,
+	setAddingToCart: ( addingToCart: boolean ) => void,
+	selectedSite: string
+): void => {
+	setAddingToCart( true );
+
+	shoppingCartManager
+		.addProductsToCart( [ fillInSingleCartItemAttributes( cartItem, productList ) ] )
+		.then( ( response: any ) => {
+			setAddingToCart( false );
+			const { errors } = response?.messages;
+			if ( errors && errors.length ) {
+				// Stay on the page to show the relevant error(s)
+				return;
+			}
+
+			page( `/checkout/${ selectedSite }` );
+		} );
+};
+
+export const recordTracksEventAddToCartClick = (
+	comparisonContext: string,
+	validatedMailboxUuids: string[],
+	mailboxesAreValid: boolean,
+	provider: string,
+	source: string,
+	userCanAddEmail: boolean,
+	userCannotAddEmailReason: any
+) => {
+	recordTracksEvent( 'calypso_email_providers_add_click', {
+		context: comparisonContext,
+		mailbox_count: validatedMailboxUuids.length,
+		mailboxes_valid: mailboxesAreValid ? 1 : 0,
+		provider: provider,
+		source,
+		user_can_add_email: userCanAddEmail,
+		user_cannot_add_email_code: userCannotAddEmailReason ? userCannotAddEmailReason.code : '',
+	} );
+};
+
+type PriceWithTermProps = {
+	className: string;
+	cost: number;
+	currencyCode: string;
+	hasDiscount: boolean;
+	sale?: number | null;
+	termLength: TermLength;
+};
+
+export const PriceWithTerm: FunctionComponent< PriceWithTermProps > = ( props ) => {
+	const { className, cost, currencyCode, hasDiscount, sale, termLength } = props;
+
+	const costClassName = `${ className }__${ hasDiscount ? 'discounted-price' : 'keep-main-price' }`;
+	const termLengthClassName = `${ className }__term`;
+	const mailboxClassName = `${ className }__mailbox`;
+	const saleClassName = `${ className }__sale`;
+
+	const showSale = sale && sale !== 0;
+
+	const lineThrough = {
+		textDecoration: hasDiscount || showSale ? 'line-through' : 'solid ',
+	};
+
+	const priceSpan = (
+		<span style={ lineThrough } className={ costClassName }>
+			{ formatCurrency( cost ?? 0, currencyCode ) }
+		</span>
+	);
+	const saleSpan = (
+		<span className={ saleClassName }>{ formatCurrency( sale ?? 0, currencyCode ) }</span>
+	);
+	const termLengthSpan =
+		termLength === 'annually' ? (
+			<span className={ termLengthClassName }>/{ translate( 'annually' ) }</span>
+		) : (
+			<span className={ termLengthClassName }>/{ translate( 'monthly' ) }</span>
+		);
+
+	const mailboxSpan = <span className={ mailboxClassName }> /{ translate( 'mailbox' ) } </span>;
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Remove mailbox' )` *ES Score: 7*